tests/ConsumerTests/Instances/InstanceTests.cs

```diff
@@ -83,7 +83,11 @@ public void DateTime()
         public void DateTimeOffset()
         {
             using var _ = new AssertionScope();
-            DateTimeOffsetInstance.iso8601_1.Value.Should().Be(new DateTimeOffset(2022, 12, 13, 0, 0, 0, TimeSpan.Zero));
+
+            // This is a bit of a hack to fix this test in none UTC timezones as the timezone is captured at compile time,
```

Comment:
Again, well spotted! I updated the wiki page with information on instances with dates and times:

https://github.com/SteveDunn/Vogen/wiki/Instances

In this case, the property is specified as a string with no time zone, so I think it's assumed to be local. I modified the assert to be:

```csharp
            DateTimeOffsetInstance.iso8601_1.Value.UtcDateTime.Should()
                .Be(DateTime.Parse("2022-12-13", null, DateTimeStyles.RoundtripKind));
```